docs/otel/sdk-metrics.md

```diff
@@ -452,5 +454,104 @@ E.g. for Java the fully qualified classname SHOULD be used in this case.
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Request Metrics
+
+### Metric: `otel.sdk.exporter.http.request.duration`
+
+This metric is [recommended][MetricRecommended].
```

Comment:
I think in general histogram health metrics should be opt-in, due to the higher storage costs of histograms.
Though for this specific metric, I think it might be debatable, as it is a very fundamental metric for network-based exporters.